test/lib.bash

```diff
@@ -100,12 +100,12 @@ function serverless_operator_kafka_e2e_tests {
   done
   kubeconfigs_str="$(array.join , "${kubeconfigs[@]}")"
 
-  RUN_FLAGS="-failfast -timeout=30m -parallel=1"
+  RUN_FLAGS=(-failfast -timeout=30m -parallel=1)
   if [ -n "${OPERATOR_TEST_FLAGS:-}" ]; then
-    RUN_FLAGS="${OPERATOR_TEST_FLAGS}"
+    IFS=" " read -r -a RUN_FLAGS <<< "$OPERATOR_TEST_FLAGS"
   fi
 
-  go_test_e2e -tags=e2e ${RUN_FLAGS} ./test/e2ekafka \
+  go_test_e2e -tags=e2e "${RUN_FLAGS}" ./test/e2ekafka \
```

Comment:
This needs to be `"${RUN_FLAGS[@]}"` . And there are more occurrences in this file